Shared/Samples/Query features with Arcade expression/QueryFeaturesWithArcadeExpressionView.swift

```diff
@@ -0,0 +1,157 @@
+                        self.error = error
+                    }
+                }
+                .overlay(alignment: .center) {
+                    if isEvaluating {
+                        VStack {
+                            Text("Evaluating")
+                            ProgressView()
+                                .progressViewStyle(.circular)
+                        }
+                        .padding()
+                        .background(.ultraThickMaterial)
+                        .cornerRadius(10)
+                        .shadow(radius: 50)
+                    }
+                }
+                .task {
+                    await mapViewProxy.setViewpointScale(2e5)
+                }
+                .errorAlert(presentingError: $error)
+        }
+    }
+}
+
+private extension QueryFeaturesWithArcadeExpressionView {
+    /// The view model for the sample.
+    class Model: ObservableObject {
+        /// A map of the "Crime in Police Beats" portal item.
+        let map: Map = {
+            // Create a portal item using a portal and item ID.
+            let portalItem = PortalItem(
+                portal: .arcGISOnline(connection: .anonymous),
+                id: .crimesInPoliceBeats
+            )
+            
+            // Create a map using the portal item.
+            let map = Map(item: portalItem)
+            return map
+        }()
+        
+        /// The Arcade evaluator for evaluating the crime count of a feature.
+        private let crimeCountEvaluator: ArcadeEvaluator = {
+            // Create a string containing the Arcade expression.
+            let expressionValue = """
+            var crimes = FeatureSetByName($map, 'Crime in the last 60 days');
+            return Count(Intersects($feature, crimes));
+            """
+            
+            // Create an Arcade expression using the string.
+            let expression = ArcadeExpression(expression: expressionValue)
+            
+            // Create an Arcade evaluator with the Arcade expression and an Arcade profile.
+            let evaluator = ArcadeEvaluator(expression: expression, profile: .formCalculation)
+            return evaluator
+        }()
+        
+        /// Evaluates the crime count for a given feature.
+        /// - Parameter feature: The ArcGIS feature evaluate.
+        /// - Returns: The evaluated crime count in the last 60 days.
+        func crimeCount(for feature: ArcGISFeature) async throws -> Double? {
+            // Create the profile variables for the script with the feature and map.
+            let profileVariables: [String: Any] = ["$feature": feature, "$map": map]
+            
+            // Evaluate for the profile variables using the evaluator.
+            let result = try await crimeCountEvaluator.evaluate(withProfileVariables: profileVariables)
+            
+            // Cast the result to get it's value.
+            let crimeCount = result.result(as: .double) as? Double
```

Comment:
From this page: https://developers.arcgis.com/arcade/function-reference/featureset_functions/#count I think it is safe to say the count is always an integer, so let's return an integer here, and adjust other places.
```swift
            let crimeCount = result.result(as: .double) as? Double ?? 0
            return Int(crimeCount)
```